src/components/ProjectCard.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const ProjectCard = (props) => {
+    const { photo, photoAlt, title, description } = props.project;
```

Comment:
The same here const `ProjectCard = (project: { photo, photoAlt, title, description }) => {` 

https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

Reply:
Handy.